app.js

```diff
@@ -21,6 +23,10 @@ app.get('/', function(req, res, next){
   }))
 })
 
+app.get('/responses', function(req, res, next){
+  res.json(smartSurveyAPICall.getData(config.apiToken, config.apiTokenSecret, config.surveyID, smartSurveyAPICall.parseResponse))
```

Comment:
What I mean it's not for this PR but if I want to render the stuff from the JSON, is it simply just render